packages/paste-core/primitives/box/src/PseudoPropStyles.ts

```diff
@@ -28,4 +28,12 @@ export const PseudoPropStyles: {[key: string]: string} = {
   _after: '&:after',
   _focusWithin: '&:focus-within',
   _placeholder: '&::placeholder',
+  _child: '& > *',
```

Comment:
Do we need any of these?

Reply:
Same, I think we might need this when someone wants to create custom radios, using RadioGroup?

Reply:
Removed and moved to `SiblingBox`